templates/cisco_ios_show_interfaces_description.template

```diff
@@ -0,0 +1,11 @@
+Value PORT (\S+)
+Value STATUS (up|down|admin down)
+Value PROTOCOL (up|down)
+Value DESCRIP (.+?)
+
+Start
+  ^Interface -> Begin
```

Comment:
```suggestion
  ^Interface\s+Status\s+Protocol\s+Description\s*$$ -> Begin
  ^\s*$$
  ^. -> Error
```
The header column will verify that the template is capturing the correct data. Another version of the OS or a different platform will have different headers, and then the template will inappropriately record the data.